packages/rocketchat-push-notifications/client/views/pushNotificationsFlexTab.js

```diff
@@ -174,21 +237,21 @@ Template.pushNotificationsFlexTab.helpers({
 	},
 	defaultAudioNotification() {
 		let preference = getUserPreference('audioNotifications');
-		if (preference === 'default' || preference == null) {
+		if (preference === 'default' || preference === null) {
```

Comment:
Why did you change this?

Reply:
It's a good practice to use the type-safe operators, [making strict comparisons](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators), to avoid type coercions. But isn't needed more, since this was removed from the code base.

Reply:
![image](https://user-images.githubusercontent.com/6303966/33906092-f4b3070c-df67-11e7-9440-4b524c7f6f48.png)
Careful, == null means it can be undefined or null, I believe that's why we kept as double instead of triple equals